src/org/zaproxy/zap/extension/fieldenumeration/FieldEnumeration.java

```diff
@@ -0,0 +1,661 @@
+	
+//	private final ExtensionHistory extension;
+        private JPanel jPanel = new JPanel();
+	private JTabbedPane tp = new JTabbedPane(); 
+	private JFrame result = new JFrame();
+	private JPanel formF = new JPanel();
+	private JPanel Panel = new JPanel();
+	private JLabel printURL = new JLabel();
+	private DefaultTableModel model = new DefaultTableModel();	
+	private JTable jTable = new JTable();
+	private JButton buttonOK = new JButton("SUBMIT");
+	private JComboBox<String> params = new JComboBox<String>();
+	private JComboBox<String> char_sets = new JComboBox<String>(); 
+	private String selected_chars  = null;
+	private JButton blacklist = new JButton("Blacklist");
+	private JButton save = new JButton("Export to CSV");
+	private GridBagConstraints c = new GridBagConstraints();
+	private GridBagConstraints c1 = new GridBagConstraints();
+	private JTextField preField = new JTextField();
+	private JTextField lowerR = new JTextField();
+	private JTextField highR = new JTextField();
+	private JTextField postField = new JTextField();
+	private Font font = new Font("Courier", Font.BOLD,12);
+
+	private ButtonGroup group = new ButtonGroup();
+	private List<JLabel> listOfLabels = new ArrayList<JLabel>();
+        private List<JTextField> listOfTextFields = new ArrayList<JTextField>();
+        private String[] num_chars = new String[] {"1", "2", "3", "4", "5"};
+        private JTextField regex = new JTextField(20);
+	private Integer indexer;
+
+	private List<NameValuePair> listParam = new ArrayList<>();
+	private HistoryReference historyRef;
+	private HttpMessage msg;
+	private String field = null;
+
+	private JTable exportTable = jTable;
+
+	/*public static void main(String args[]) {
+		new FieldEnumeration();
+	}*/
+	
+	public FieldEnumeration() throws HeadlessException {
+		super();
+		initialize();
+	}
+	
+	public FieldEnumeration(Frame arg0, boolean arg1) throws HeadlessException {
+      		super(arg0, arg1);
+        	initialize();
+       }
+
+	private void persistAndShowMessage(final HttpMessage httpMessage) {
+    		if (!EventQueue.isDispatchThread()) {
+        		EventQueue.invokeLater(new Runnable() {
+	            @Override
+        	    public void run() {
+                	persistAndShowMessage(httpMessage);
+            	    }
+        	});
+        	return;
+    		}
+ 
+    		try {
+        		Session session = Model.getSingleton().getSession();
+        		HistoryReference ref = new HistoryReference(session, HistoryReference.TYPE_ZAP_USER, httpMessage);
+        		final ExtensionHistory extHistory = Control.getSingleton().getExtensionLoader().getExtension(ExtensionHistory.class);
+        			if (extHistory != null) {
+            				extHistory.addHistory(ref);
+        			}
+        		SessionStructure.addPath(Model.getSingleton().getSession(), ref, httpMessage);
+    		} catch (HttpMalformedHeaderException | DatabaseException e) {
+        		LOGGER.warn("Failed to persist message sent:", e);
+    		}
+	}
```

Comment:
instead of continuing to keep listing one-off issues. I'm going to suggest that since this is all new content you just auto format it.

In exclipse you can use <kbd>ctrl</kbd> + <kbd>shift</kbd> + <kbd>F</kbd>
http://help.eclipse.org/oxygen/index.jsp?topic=%2Forg.eclipse.jdt.doc.user%2Freference%2Fref-menu-source.htm

Reply:
Apologies for repeated mistakes!